### PR TITLE
feat: [Chaos-151]: ChaosDelegate creation using connectors (UI)

### PR DESCRIPTION
**Pull request by vanshBhatia-A4k9**
Signed-off-by: Vansh Bhatia <vansh.bhatia@harness.io>

### Summary

- passing the useCreateConnectorModal to the CHAOS MFE

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 2 Issues 
* Code issue 2 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10456)

**Review by SahilKr24 — DISMISSED** [review]
LGTM!

**Review by hrishavjha — APPROVED** [review]

**Review by arkajyotiMukherjee — DISMISSED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by SahilKr24 — DISMISSED** [review]

**Review by hrishavjha — APPROVED** [review]

File: src/modules/75-chaos/RouteDestinations.tsx [hunk 1]
```diff
@@ -547,7 +548,8 @@ export default function ChaosRoutes(): React.ReactElement {
             ConnectorReferenceField,
             OverviewChartsWithToggle,
             validateYAMLWithSchema,
-            NavigationCheck
+            NavigationCheck,
+            useCreateConnectorModal
```
Comment (abhinavrastogi-harness) [inline src/modules/75-chaos/RouteDestinations.tsx:552]: hooks should be passed in a separate key `customHooks`.
`customComponents` should be used only for passing components. that just makes more semantic sense.

**Review by abhinavrastogi-harness — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by hrishavjha — DISMISSED** [review]

**Review by SahilKr24 — DISMISSED** [review]

**Comment by arkajyotiMukherjee** [comment]
One check is failing with the message `No Addition allowed in src/strings/strings.en.yaml. Please use module level string files instead`. But the file changes doesn't include string files @abhinavrastogi-harness

**Comment by arkajyotiMukherjee** [comment]
retrigger typecheck

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**